bedrock/firefox/views.py

```diff
@@ -406,7 +407,16 @@ def get_template_names(self):
         return [template]
 
 
-class WhatsnewView(l10n_utils.LangFilesMixin, TemplateView):
+class WhatsnewView(VariationMixin, l10n_utils.LangFilesMixin, TemplateView):
+    template_context_variations = ['a', 'b', 'c']
+    # locales *not* targeted by experiment
+    excluded_locales = ('de', 'en-GB', 'en-US', 'en-ZA', 'es-ES', 'fr', 'id',
+                        'ja', 'ja-JP-mac', 'pl', 'pt-BR', 'ru', 'zh-CN',
+                        'zh-TW')
+    # calculate non-excluded locales
+    # intent is to find locales in PROD_LANGUAGES *not* present in excluded_locales
+    variation_locales = list(set(settings.PROD_LANGUAGES).symmetric_difference(set(excluded_locales)))
```

Comment:
This will work, but it's a bit more confusing than what you really want I think. The following should be sufficient. 

```python
excluded_locales = {'de', 'en-GB', 'en-US', 'en-ZA', 'es-ES', 'fr', 'id',
                    'ja', 'ja-JP-mac', 'pl', 'pt-BR', 'ru', 'zh-CN',
                    'zh-TW'}
variation_locales = list(set(settings.PROD_LANGUAGES) - excluded_locales)
```

Note the curly braces around the `excluded_locales` value, that's a set literal, so it's already a set.